### DISCUSSION TITLE
Creality UI, KAMP not working

### DISCUSSION THREAD

**Discussion by negativ72rus** [Guilouz/Creality-Helper-Script-Wiki#605] [Issues]
Hi!
If i start printing g-code from the printer screen, the adaptive table map does not work.
With the "calibration" checkbox checked, the full table map is launched. If you do not check this box, the table height map is not built at all.
K1C
Standard UI Creality.

**Comment by Guilouz** [comment]
It’s a known issue, you must start print from web ui to use kamp.

**Reply by Jesus5431** [reply]
Has this been fixed especially for the ender 3 v3 printers as I like starting prints from the screen
